db/migrate/20201212043140_create_shares.rb

```diff
@@ -0,0 +1,10 @@
+class CreateShares < ActiveRecord::Migration[6.0]
+  def change
+    create_table :shares do |t|
+      t.integer :profile_shared
+      t.integer :status
```

Comment:
Esse status precisa trocar pra bool e virar um "shared' pra pessoa poder "ativar/desativar" o compartilhamento de perfil e tal

Reply:
Entendi, vou ajustar!

Reply:
Sim, seria algo assim, eu quis dizer em trocar o nome de "status" para "shared" e virar bool, porque ai ele esta "compartilhado" ou não, pelo true ou false, mas pode ser também active inactive, o funcionamento é o mesmo, mas como vai ser ou um ou outro, faz mais sentido ser bool, mas se for facilitar pode deixar enum mesmo

Reply:
De qualquer forma, faltou declarar o enum no model pra ativo e inativo como comentou

Reply:
entendi, mas a palavra shared da conflito com o o banco de dados! Acredito que seja uma palavra reservada do Rails

Reply:
Estou subindo o PR com este ajuste para fazer o code review